### PR TITLE
[authpolicy-v2] Well-known attributes in the generated AuthConfigs

### PR DESCRIPTION
**Pull request by guicassolato**
Closes:
- #265

### CODE REVIEW

**Review by Boomatang — COMMENTED** [review]
The changes seem good. Do you have an example verification steps that could be done manually?

**Comment by guicassolato** [comment]
> Do you have an example verification steps that could be done manually?

Same ones from #256 will do. You should look for the diff in the generated AuthConfigs.

File: controllers/authpolicy_controller_test.go [hunk 1]
```diff
@@ -770,7 +770,7 @@ var _ = Describe("AuthPolicy controller", func() {
...
...any":[{"selector":"auth.identity.admin","operator":"eq","value":"true"},{"selector":"auth.metadata.user-groups","operator":"incl","value":"privileged"}]}]}}},"response":{"unauthenticated":{"message":{"value":"Missing verified JWT injected by the gateway"}},"unauthorized":{"message":{"value":"User must be admin or member of privileged group"}},"success":{"headers":{"x-username":{"when":[{"selector":"request.headers.x-propagate-username.@case:lower","operator":"matches","value":"1|yes|true"}],"plain":{"value":null,"selector":"auth.identity.username"}}},"dynamicMetadata":{"x-auth-data":{"when":[{"patternRef":"authz-and-rl-required"}],"json":{"properties":{"groups":{"value":null,"selector":"auth.metadata.user-groups"},"username":{"value":null,"selector":"auth.identity.username"}}}}}}},"callbacks":{"unauthorized-attempt":{"when":[{"patternRef":"authz-and-rl-required"},{"selector":"auth.authorization.admin-or-privileged","operator":"neq","value":"true"}],"http":{"url":"http://events/unauthorized","method":"POST","body":{"value":null,"selector":"\\{\"identity\":{auth.identity},\"request-id\":{request.id}\\}"},"contentType":"application/json","credentials":{"authorizationHeader":{}}}}}}`))
```
Comment (maleck13) [inline controllers/authpolicy_controller_test.go:773]: how are we maintining this?

**Review by maleck13 — APPROVED** [review]
/lgtm

**Review by didierofrivia — APPROVED** [review]
👍🏼

**Pull request merged**